bindings/python/multibody/geometry-data.hpp

```diff
@@ -44,67 +44,68 @@ namespace pinocchio
         bp::class_< std::vector<CollisionPair> >("StdVec_CollisionPair")
         .def(bp::vector_indexing_suite< std::vector<CollisionPair> >());
       }
-      
-      
     }; // struct CollisionPairPythonVisitor
 
     struct GeometryDataPythonVisitor
       : public boost::python::def_visitor< GeometryDataPythonVisitor >
     {
       
-    
-
       /* --- Exposing C++ API to python through the handler ----------------- */
       template<class PyClass>
       void visit(PyClass& cl) const
       {
         cl
-        .def(bp::init<GeometryModel>(bp::arg("geometry_model"), "Default constructor from a given GeometryModel"))
+        .def(bp::init<GeometryModel>(bp::args("self","geometry_model"),
+                                     "Default constructor from a given GeometryModel"))
         
         .def_readonly("oMg",
                       &GeometryData::oMg,
-                      "Vector of collision objects placement relative to the world.")
+                      "Vector of collision objects placement relative to the world frame.\n"
+                      "note: These quantities have to be updated by calling updateGeometryPlacements.")
+        
 #ifdef PINOCCHIO_WITH_HPP_FCL
         .def_readonly("activeCollisionPairs",
                       &GeometryData::activeCollisionPairs,
                       "Vector of active CollisionPairs")
         .def_readonly("distanceRequest",
                       &GeometryData::distanceRequest,
-                      "Defines which information should be computed by fcl for distances")
+                      "Defines which information should be computed by FCL for distance computations")
         .def_readonly("distanceResults",
                       &GeometryData::distanceResults,
                       "Vector of distance results.")
         .def_readonly("collisionRequest",
                       &GeometryData::collisionRequest,
-                      "Defines which information should be computed by fcl for collisions.")
+                      "Defines which information should be computed by FCL for collision computations.")
```

Comment:
A note for the future:

it is possible to define a *security_margin* and a *break_distance* for a collision request. Most likely, for robotics application, those threshold will be different for each collision pairs. E.g. the two hands can have a large security margin while the two hips cannot. It may be a good idea to have as many collision requests as collision pairs.

Reply:
I will add this note for additional information.